### DISCUSSION TITLE
Lookup - Comportamento inesperado

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1118] [Tire suas dúvidas] [answered]
Creio que o componente lookup esta com problema, eu fiz uma simulação do problema no próprio portal e usei o exemplo básico, segue abaixo as instruções para a simulação:

1. Clique no ícone de pesquisa para abrir o modal;
2. Marque um item da lista apresentada no modal;
3. Clique no botão Selecionar;
4. No componente se mostrado a informação selecionada;
5. Vamos repetir o processo;
6. Clique no ícone de pesquisa para abrir o modal;
7. Será apresentado uma lista com o item correspondente ao componente lookup já marcado;
8. **Não troque o item marcado;**
9. **Apenas clique no botão selecionar;**
10. **O conteúdo do componente deveria permanecer o mesmo, porem o mesmo é apagado!**

**Comment by alinelariguet** [comment, marked as answer]
Obrigada por reportar, isso é realmente um bug, estou convertendo essa discussão para uma issue.

**Reply by wsteixeira** [reply]
Obrigado pelo retorno.
